src/main/java/lotto/LottoApplicationController.java

```diff
@@ -0,0 +1,71 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoApplicationController {
+	private PurchaseAmount purchaseAmount;
+	private WinningNumbers winningNumbers;
+	private LottoStatistics lottoStatistics = new LottoStatistics();
+	private List<Lotto> lottos = new ArrayList<>();
+
+	public boolean enterPurchaseAmount() {
+		try {
+			purchaseAmount = new PurchaseAmount(InputView.enterPurchaseAmount());
+		} catch (IllegalArgumentException exception) {
+			OutputView.printMessage(exception.getMessage());
+			return true;
+		} catch (IllegalStateException exception) {
+			OutputView.printMessage(exception.getMessage());
+			return true;
+		}
+		return false;
+	}
+
+	public void printPurchaseQuantity() {
+		int purchaseQuantity = purchaseAmount.value() / CommonConstant.LOTTO_SALES_PRICE;
```

Comment:
purchaseAmount의 상태를 확인하고 직접 계산하는 것보다 객체에게 메시지를 던져보는 것은 어떨까요 ?
```suggestion
		int purchaseQuantity = purchaseAmount.divide(CommonConstant.LOTTO_SALES_PRICE);
```